modules/extensions-support/src/main/resources/META-INF/services/org.mule.extension.introspection.declaration.spi.ModelEnricher

```diff
@@ -0,0 +1 @@
+org.mule.module.extension.internal.runtime.connector.ConnectorModelEnricher
```

Comment:
Once there are more model enrichers is it the order in this text file that determines the order in which they should be applied?